suite/tests/CMakeLists.txt

```diff
@@ -3722,6 +3737,15 @@ if (BUILD_CLIENTS)
     torunonly_drcacheoff(func_view common.fib "-record_function fib|1"
       "@-simulator_type@func_view" "only_5")
 
+    if (DR_HOST_X86 AND DR_HOST_X64 AND LINUX)
+      # Requires sudo to access pagemap.
+      # XXX: Should we not enable this outside of the Github suite where we know
+      # we have passwordless sudo?  The pause for a password may cause problems
```

Comment:
So today it will pause? Should we solve that in this PR itself, or else users might see issues with ctest?

Reply:
Per offline discussion, added a new option RUN_SUDO_TESTS and made this test not run unless it is set, which it is for automated_ci.